artman/cli/main.py

```diff
@@ -69,7 +69,8 @@ def main(*args):
                 pipeline.flow, engine='serial', store=pipeline.kwargs)
             engine.run()
         except:
-            logger.fatal(traceback.format_exc())
+            logger.error(traceback.format_exc())
+            sys.exit(128)
```

Comment:
Error exit codes are [recommended](https://docs.python.org/2/library/sys.html#sys.exit) to be in the 1-127 range.

Reply:
Good catch. Fixed.